app/src/api.ts

```diff
@@ -0,0 +1,82 @@
+/** urls for this github repo */
+
+export const repo = "3b1b/captions";
+export const repoFull = `https://github.com/${repo}`;
+export const repoRaw = `https://raw.githubusercontent.com/${repo}`;
+export const website = "https://3blue1brown.com";
+export const websiteRaw =
+  "https://raw.githubusercontent.com/3b1b/3Blue1Brown.com";
+export const cloudFuncBase =
+  "https://us-central1-mystic-fountain-606.cloudfunctions.net/test-function";
```

Comment:
Needs to be updated to a permanent one under your account. Temp GCP endpoint I set up for now.